spec/features/user_views_vendor_list_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'user views vendor list', %{
+  As visitor
+  I want to view a list of all available food trucks/vendors
+  So that I can decide what food truck/vendors I want to visit
+} do
+  # Acceptance Criteria
+
+  # * As a visitor
+  # * When I visit the home page
+  # * I should be able to click on a link to view all vendors.
+
+  # * As a visitor
+  # * When I visit the vendors page
+  # * I should be able to see the list of vendors.
+
+  scenario 'visitor can view the list of vendors via the vendors link' do
+    visit vendors_path
```

Comment:
Should we really be visiting the `vendors_path` yet? Or just the home page to start with? (If they're the same, still use the syntax `root_path`, assuming you have that defined - it just makes it more clear what you're trying to do)